apps/worker/src/app/workflow/workflow.module.ts

```diff
@@ -63,6 +67,16 @@ import { SharedModule } from '../shared/shared.module';
 
 const REPOSITORIES = [JobRepository];
 
+export const getIsTopicNotificationEnabled = {
+  provide: GetIsTopicNotificationEnabled,
+  useFactory: async (featureFlagsServiceItem: FeatureFlagsService): Promise<GetIsTopicNotificationEnabled> => {
+    const useCase = new GetIsTopicNotificationEnabled(featureFlagsServiceItem);
+
+    return useCase;
+  },
+  inject: [FeatureFlagsService],
+};
+
```

Comment:
You don't need to do this as you can just import the class reference `GetIsTopicNotificationEnabled`. The instantiation is already done inside custom providers in application generic.